Library/Homebrew/download_strategy.rb

```diff
@@ -502,6 +492,22 @@ def _fetch
 
     curl_download s3url, to: temporary_path
   end
+
+  private
+
+  def load_aws_sdk
+    retries = 0
+    aws_sdk_gem = "aws-sdk-v1"
+    aws_sdk_gem_exe = "aws-rb"
+    begin
+      require aws_sdk_gem
+    rescue LoadError
+      raise if retries.positive?
+      Homebrew.install_gem_setup_path! aws_sdk_gem, HOMEBREW_AWS_SDK_V1_VERSION, aws_sdk_gem_exe
```

Comment:
You should be able to just always use `install_gem_setup_path!`
 — that’s how we do it everywhere else. Would make this method much simpler.

Reply:
@alyssais Thanks! I did switch to that approach before I discovered I had to abandon the use of the method entirely :(